judge/forms.py

```diff
@@ -284,3 +285,17 @@ def clean_key(self):
         if Contest.objects.filter(key=key).exists():
             raise ValidationError(_('Contest with key already exists.'))
         return key
+
+
+class ProblemPointsVoteForm(ModelForm):
+    class Meta:
+        model = ProblemPointsVote
+        fields = ['points', 'note']
+        # The default 'required' error message is triggered if non-integer data is sent even if
+        # the user enters data into the field, due to the form serialization replacing the invalid
+        # data with an empty string. It is overridden here to provide a suitable response.
+        error_messages = {
+            'points': {
+                'required': _('Please enter an integer.')
```

Comment:
```suggestion
                'required': _('Please enter an integer.'),
```

`./judge/forms.py:299:58: C812 missing trailing comma`